core/arch/arm/tee/tee_rpmb.c

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2016, Linaro Limited
```

Comment:
I agree that it's small, but this file has been changed recently in other patches too so the copyright year should be updated.
I can add another year at the ST line instead of that's preferred.